node/narwhal/src/primary.rs

```diff
@@ -248,6 +259,7 @@ impl<N: Network> Primary<N> {
             for signature in signatures.keys() {
                 stake = stake.saturating_add(self.committee.get_stake(&signature.to_address()));
             }
+            debug!("Stake {stake}/{}", self.committee.quorum_threshold()?);
```

Comment:
nit: Switch to `trace!`

Reply:
Actually, on second thought, I think this is okay, it could really go either way. Up to you!